.eslintrc.js

```diff
@@ -4,7 +4,32 @@ module.exports = {
   extends: ['custom'],
   settings: {
     next: {
-      rootDir: ['apps/*/', 'packages/ui/*/', 'packages/onboard-helpers/*/'],
-    },
+      rootDir: ['apps/*/', 'packages/ui/*/', 'packages/onboard-helpers/*/']
+    }
   },
+  overrides: [
+    // enforce Feature Sliced rules for loan app
+    {
+      files: [
+        'apps/loan/src/{app,processes,pages,widgets,features,entities,shared}/**/*.{ts,tsx}',
```

Comment:
```suggestion
        'apps/loan/src/{app,pages,widgets,features,entities,shared}/**/*.{ts,tsx}',
```
Processes is a deprecated layer